exchange_server/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # CHANGELOG - exchange_server
 
+## 1.2.0 / 2018-10-12
+
+* [Added] Pin pywin32 dependency. See [#2322](https://github.com/DataDog/integrations-core/pull/2322).
+
```

Comment:
This doesn't include the port removal for pdh checks?

Reply:
Port never did anything

Reply:
It didn't have it